cmd/conf/init.go

```diff
@@ -48,6 +49,7 @@ var InitConfigDefault = InitConfig{
 	DevInitAddress:           "",
 	DevInitBlockNum:          0,
 	Empty:                    false,
+	ExtractWasm:              false,
```

Comment:
I don't like the name. Extract is not the important action..
Maybe import-wasm?